src/main/resources/db/changelog/changes/5-ays-dml.xml

```diff
@@ -63,6 +63,13 @@
                     '77ece256-bf0e-4bbe-801d-173083f8bdcf', 'In Progress Assignment', 'Ahmet', 'Mehmet',
                     '90', '1234567890', ST_GeomFromText('POINT(40.981210 29.000000)', 4326),
                     'IN_PROGRESS', 'AYS', CURRENT_TIMESTAMP);
+
+            INSERT INTO AYS_ASSIGNMENT (ID, USER_ID, INSTITUTION_ID, DESCRIPTION, FIRST_NAME, LAST_NAME, COUNTRY_CODE,
```

Comment:
var olan changelog'u güncellemek yerine yeni bir tane oluşturalım. Hali hazırdaki uygulamayı patlatabilir.

Reply:
Neden hali hazırdaki uygulamayı patlatsın ki? Database'e yeni veri ekleniyor sadece. Anlamadım.
Changeloglar'da kullanıcıların eklendiği, assignmentların eklendiği vs. gibi bölümler var. Ben de uygun olan yerlerde ekleme yaptım.

Reply:
Spring boot açıldığında, liquibase changelog scriptleri sıra ile çalıştırıyor. Çalışan script bilgileri databasede özel bir tabloda aşağıdaki örnekteki gibi tutuluyor. Eğer bu dosyalar güncellenir ise `MD5SUM` kolonu değişecek ve çalıştırılmış bir script değiştiği için hata alıcaktır. Bu durum ECS'e geçmeden önce de yaşanıyordu. @agitrubard changelogların değiştiği deploymentlarda database'i sıfırlıyordu.

| ID | AUTHOR | FILENAME | DATEEXECUTED | ORDEREXECUTED | EXECTYPE | MD5SUM | DESCRIPTION | COMMENTS | TAG | LIQUIBASE | CONTEXTS | LABELS | DEPLOYMENT\_ID |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| 1-ays-ddl | agitrubard | db/changelog/changes/1-ays-ddl.xml | 2023-11-03 17:10:48 | 1 | EXECUTED | 9:335b0bef24107269477b625586bbe408 | createTable tableName=AYS\_INSTITUTION; sql; createTable tableName=AYS\_ADMIN\_USER; sql; createTable tableName=AYS\_ADMIN\_USER\_REGISTER\_VERIFICATION; sql; createTable tableName=AYS\_USER; sql; createTable tableName=AYS\_PARAMETER |  | null | 4.23.2 | null | null | 9031448711 |

Reply:
Düzelttim.